CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ## [Unreleased] - TBD
 ### Fixed
 - Fixed CEDS HEMCO_Config.rc entries to emit TMB into the TMB species (and not HCOOH)
+- CEDS 0.1 x 0.1 degree emissions (in `HEMCO/CEDS/v2024-06`) are now the default instead of CEDSv2
```

Comment:
I think this update should be "Added" not "Fixed". Also, in the changed section it could say the new 0.1x0.1 CEDS is the new default, replacing 0.5x0.5 CEDS

Reply:
So in Added section: "Added CEDS 0.1 x 0.1 degree emissions (in `HEMCO/CEDS/v2024-06`)"
And in Change section: "Updated default CEDS from CEDSv2 (0.5x0.5) to new CEDS at 0.1x0.1"

Reply:
Now fixed in 0fd18bd